qiskit/compiler/transpiler.py

```diff
@@ -88,22 +88,22 @@ def transpile(
 ) -> Union[QuantumCircuit, List[QuantumCircuit]]:
     """Transpile one or more circuits, according to some desired transpilation targets.
 
-    All arguments may be given as either a singleton or list. In case of a list,
-    the length must be equal to the number of circuits being transpiled.
+    .. deprecated:: 0.23.0
+
+        All arguments (except for ``circuits``) may be given as either a singleton or list. In
+        case of a list, the length must be equal to the number of circuits being transpiled. The
```

Comment:
> All arguments (except for ``circuits``) may be given as either a singleton or list. In case of a list, the length must be equal to the number of circuits being transpiled.

Is there a way to make it clear that this is a quote from the original documentation, and that it is actually the thing being deprecated? Not sure if there's a rule where we must include the deprecated documentation verbatim, but it could be helpful to reword this to make it clear that we're not describing the new functionality, but rather quoting what is no longer applicable.

Reply:
I can reword it, I was just being lazy and saw this block in the docs I just updated it to basically say "... and it's deprecated", I'll clean this up.